package.json

```diff
@@ -24,6 +24,7 @@
     "email": "clemens@bitcoincomputer.io"
   },
   "private": true,
+  "packageManager": "npm@10.2.4",
```

Comment:
Do we need to specify the exact npm version?

Reply:
no we don't need the exact version, but turbo requires it. I added my latest version.